PayloadPkg/OsLoader/KeyManagement.c

```diff
@@ -256,10 +256,10 @@ RpmbKeyProvisioning (
   }
 
   // Get manufacturing state
-  Eom = (LoaderPlatformInfo->SecurityFlags & BIT2) >> BIT1;
+  Eom = ((LoaderPlatformInfo->HwState & HWSTATE_MANUFACTURING) != 0) ?TRUE:FALSE;
```

Comment:
By comments, looks it should be "== 0" instead of "!= 0". 
Better rename it to HWSTATE_IN_MANUFACTURING_MODE to make more clear?

Reply:
I haven't interpreted HWSTATE_MANUFACTURING definition correctly. Thanks. I would fix it.